HousingSearchApi/Startup.cs

```diff
@@ -18,8 +18,10 @@
 using Microsoft.OpenApi.Models;
 using HousingSearchApi.V1.Gateways;
 using HousingSearchApi.V1.Infrastructure;
+using HousingSearchApi.V1.Infrastructure.Sorting;
 using HousingSearchApi.V1.UseCase;
```

Comment:
let's remember to remove all the unused use cases and their related codes which were part of the base-api

Reply:
I run a remove all unused every time I push!